src/App.js

```diff
@@ -278,8 +278,8 @@ const App = () => {
                           </div>
                         )}
                         <p className="description-text" style={{ textAlign: 'left', paddingLeft: '10%', paddingRight: '10%' }}>
-                            Before running the model/optimizer, all parameters must be set.​ Either insert parameters manually, use default settings or upload a CSV-File in the same format as stated in XX  
-                            <a href={MyPDF} target="_blank" rel="noopener noreferrer">here</a> to generate parameters automatically.​
+                            Before running the model/optimizer, all parameters must be set.​ Either insert parameters manually, use default settings or upload a CSV-File in the same format as stated in Table C.5 in   
+                             <a href={MyPDF} target="_blank" rel="noopener noreferrer"> here</a> to generate parameters automatically.​
```

Comment:
super nitpicky, aber mich stört es ein bisschen, dass das Leerzeichen vor "here" auch schon Teil vom link ist. Hab mal ausprobiert: wenn du die zwei Zeilen auf eine Zeile schreibst, das Leerzeichen zwischen "...C.5 in" und "<a href..." machst, und das Leerzeichen vor "here" wegmachst, dann siehts schöner aus :)

Reply:
hahah ok Lösung ist also das in eine einzelne Zeile zu schreiben :D hatte die ganze Zeit schon mit anderem rumprobiert